.castor/utils.php

```diff
@@ -98,15 +98,18 @@ function create_default_context(): Context
         $data['user_id'] = 1000;
     }
 
-    // @phpstan-ignore-next-line
-    return new Context($data, pty: 'dev' === $data['env']);
+    return new Context(
+        // @phpstan-ignore-next-line
+        $data,
+        pty: 'dev' === $data['env'],
+        environment: create_default_environment(),
+    );
 }
 
 function docker_compose_run(
     string $runCommand,
     Context $c = null,
     string $service = 'builder',
-    string $user = 'app',
```

Comment:
It's not possible to exec something with something else than `app`. Anyway, we never changed this value, right?